open_city_profile/settings.py

```diff
@@ -37,10 +39,25 @@
 if os.path.exists(env_file):
     env.read_env(env_file)
 
+# Extract version (or commit hash) for sentry
 try:
-    version = raven.fetch_git_sha(checkout_dir())
+    version = subprocess.check_output(["git", "describe"]).strip()
```

Comment:
you could do `git describe --always`, it will fall back on the abbreviated commit hash, which is probably enough for sentry.

Reply:
would help you avoid nested try/except clauses 🙂

Reply:
Excellent point, much better way to do it. Will update.